src/lib/radio/radio.html

```diff
@@ -1,6 +1,6 @@
 <!-- TODO(jelbourn): render the radio on either side of the content -->
 <!-- TODO(mtlin): Evaluate trade-offs of using native radio vs. cost of additional bindings. -->
-<label [attr.for]="inputId" class="md-radio-label">
+<label [attr.for]="inputId" class="md-radio-label" [class.md-ripple-focused]="_isFocused">
```

Comment:
Can you add a TODO to only show this focus indicator when the focus came from a keyboard event?